Springer Link.js

```diff
@@ -196,13 +197,17 @@ function complementItem(doc, item) {
 			item.abstractNote = abs.trim();
 		}
 		item.abstractNote = titleTextGerman + "\n\n" + ZU.trimInternal(abs).replace(/^Abstract[:\s]*/, "");
-		if (item.abstractNote === null) item.abstractNote = '';
-	}
+    }
+    if (!item.abstractNote)
+        item.abstractNote = '';
 
-	let tags = ZU.xpathText(doc, '//span[@class="Keyword"] | //*[contains(concat( " ", @class, " " ), concat( " ", "c-article-subject-list__subject", " " ))]//span');
+	let tags = ZU.xpathText(doc, '//span[@class="Keyword"] | //*[contains(concat( " ", @class, " " ), concat( " ", "c-article-subject-list__subject", " " ))]//span | \
```

Comment:
nested `//` may result in bad performance

Reply:
Yes but this was the originally present expression and is performance really an issue in this context?

Reply:
i guess its fine if you dont experience any problems